lib/qless/server.rb

```diff
@@ -20,6 +20,8 @@ class Server < Sinatra::Base
 
     set :erb, escape_html: true
 
+    set :show_exceptions, false
```

Comment:
sinatra docs say this thing is `true` only in development environment 😕 
```
show_exceptions
Show a stack trace in the browser when an exception happens. Enabled by default when environment is set to "development", disabled otherwise.
```

Reply:
I'm looking into how the environment stuff works; just did this for now as a quick fix.